src/Gui/CMakeLists.txt

```diff
@@ -52,10 +52,12 @@ include_directories(
     ${EIGEN3_INCLUDE_DIR}
     ${COIN3D_INCLUDE_DIRS}
     ${PYTHON_INCLUDE_DIRS}
+    ${OCC_INCLUDE_DIR}
```

Comment:
Please DO NOT any OCC stuff to the core system!

Reply:
It is semi mentioned in the PR description:
> For now it adds dependency on Part to the Gui module but it will be removed later when I figure out how to do it properly.

If you have any suggestion on how to skip that dependency it would be nice. I don't see any good way of providing the ability to support choosing the transform origin without relying on Part. My current plan is to simply utilize newly created `getTransformDialog` method and override it when Part is available, but maybe there is a better way.

Reply:
It's a virtual method in ViewProviderDragger and you can override it for the view providers from PartGui.